lighthouse-core/test/gather/gatherers/manifest-test.js

```diff
@@ -47,6 +47,46 @@ describe('Manifest gatherer', () => {
     });
   });
 
+  it('throws an error when manifest is saved as BOM UTF-8', () => {
+    const fs = require('fs');
+    const manifestWithoutBOM = fs.readFileSync(__dirname + '/../../fixtures/manifest.json')
+      .toString();
+    const manifestWithBOM = fs.readFileSync(__dirname + '/../../fixtures/manifest-bom.json')
+      .toString();
+
+    const getDriver = (data) => {
+      return {
+        driver: {
+          getAppManifest() {
+            return Promise.resolve({
+              data,
+              errors: [],
+              url: EXAMPLE_MANIFEST_URL
+            });
+          }
+        },
+        url: EXAMPLE_DOC_URL
+      };
+    };
+
+    const promises = [];
+    promises.push(manifestGather.afterPass(getDriver(manifestWithBOM))
```

Comment:
damn forgot to change the test text as we throwed before it should say pass when manifest has BOM